galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager.ivt/src/main/java/dev/galasa/core/manager/ivt/CoreManagerIVT.java

```diff
@@ -51,4 +63,27 @@ public void checkCoreManager() throws Exception {
         logger.info("The Core Manager field has been correctly initialised");
     }
 
+    @Test
+    public void checkCreateTestProperty() {
+    	assertThat(prop).isEqualTo("TEST");
```

Comment:
How is this property being set?  Did you add something into your own CPS or overrides?

Reply:
I put it in my overrides.

Reply:
Ok so ideally this should be in the global CPS for our ecosystem otherwise the test will fail in automation. Can you add that in please